README.md

```diff
@@ -28,6 +28,23 @@ Using Yarn:
 yarn add nucleojs
 ```
 
+## Documentation
+
```

Comment:
An bad experience I have seeing these documentation links in README's is that you don't know it's moving you to another page, I think always it's going just to anchor me to some part of this page. What about we tell people about this?
```suggestion
Above are links that will take you to our `API_DOCUMENTATION.md` file present in this repository with deeper information and documentation to Nucleo usage.
```